boards/librem_l1um/librem_l1um.config

```diff
@@ -32,7 +32,7 @@ export CONFIG_TPM=y
 export CONFIG_BOOTSCRIPT=/bin/gui-init
 export CONFIG_BOOT_REQ_HASH=n
 export CONFIG_BOOT_REQ_ROLLBACK=n
-export CONFIG_BOOT_KERNEL_ADD="intel_iommu=on"
+export CONFIG_BOOT_KERNEL_ADD="intel_iommu=oncpufreq=xen:hwp"
```

Comment:
This needs a space assuming we go ahead with the change (thoughts in review summary)